cmd/broker/binding_test.go

```diff
@@ -49,4 +51,16 @@ func TestBinding(t *testing.T) {
 	b, _ = io.ReadAll(respRuntimes.Body)
 	suite.Log(string(b))
 	suite.Log(resp.Status)
+
+	t.Run("should return 400 for incorrect parameters", func(t *testing.T) {
```

Comment:
Please add comment or change name to clarify, that incorrect means string instead of int.